app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,58 @@
+module OrdersHelper
+  def us_states
```

Comment:
Since this information isn't going to change, it might be better to make these part of a constant instead of a method.